CONTRIBUTING.md

```diff
@@ -23,6 +23,22 @@ source code by running the following make target in your build directory:
 $ make format
 ```
 
+### Test coverage
+
+Make sure changes are covered by tests.
+Code coverage instrumentation can be enabled using a build option and
+a detailed html report can be viewed using following example:
+
+```sh
+$ cmake -DENABLE_COVERAGE=ON ..
+$ make all test coverage
+$ xdg-open ./coverage.html
+```
```

Comment:
Perhaps add `mkdir -p build; cd build`, just so everyone understands how to use CMake.

Reply:
Ah, yes Ill add that